cmd/monaco/runner/runner.go

```diff
@@ -49,7 +50,9 @@ func RunCmd(fs afero.Fs, command *cobra.Command) error {
 			}
 		}
 	}()
-	err := command.Execute()
+
+	ctx = context.WithValue(ctx, support.SupportArchive{}, supportArchivePtr)
```

Comment:
The key should not be set like this, please follow the best practices for defining context keys and setting/receiving context values.
After quick Google search: https://mainfunction.org/articles/golang-context-with-value-best-practices/

This article also gives information on why: https://medium.com/@sanhdoan/go-context-best-practices-a-better-way-to-store-values-3c1fb3bf4e84

Reply:
done

Reply:
Now two different struct are used, one as key, second as value holder